templates/zerver/features.html

```diff
@@ -64,6 +64,13 @@ <h3>CUSTOMIZABLE AUTOMATIC LINKIFICATION</h3>
                     “#1235” or a commit ID.
                 </p>
             </div>
+            <div class="feature-block">
+                <h3>EXTERNAL CODE PLAYGROUND OPTIONS</h3>
+                <p>
+                    Configure playground options to open code in a Zulip
+                    message's code block using external playgrounds.
```

Comment:
This description is probably going to be too opaque for a random user.  It's possible we just want to this advertised on /for/open-source and on the "code blocks" section of the markdown advice, since it's a little hard to explain and I'm not sure how much non-OSS organizations will want this.

@alya thoughts?

Reply:
Hmm, good question... Are you suggesting not having a way to set it up in the menus at all?

In general, I think we should make sure we never use the term "playground" by itself, as it would be unfamiliar and confusing to many users. "Code playground" at least makes it clear for non-coders that "it's a thing for code that I don't care about".

I suppose we could also make the menu option more generically say "Code blocks". I think it would be less weird for users who are unfamiliar with playgrounds, though more opaque for users who are.

Reply:
No, I was suggesting we not try to explain this feature on /features (aka that specific landing page).

I think a "Code blocks" title here with the text explaining the "Copy code block content" button and external playgrounds would be reasonable.

Reply:
Cool! Updated the /features page with a "Code blocks" section.